homeassistant/generated/integrations.json

```diff
@@ -1581,57 +1581,62 @@
     "google": {
       "name": "Google",
       "integrations": {
-        "google": {
-          "config_flow": true,
+        "google_assistant": {
+          "config_flow": false,
+          "iot_class": "cloud_push",
+          "name": "Google Assistant"
+        },
+        "google_cloud": {
+          "config_flow": false,
+          "iot_class": "cloud_push",
+          "name": "Google Cloud Platform"
+        },
+        "google_domains": {
+          "config_flow": false,
           "iot_class": "cloud_polling",
-          "name": "Google Calendar"
+          "name": "Google Domains"
+        },
+        "google_maps": {
+          "config_flow": false,
+          "iot_class": "cloud_polling",
+          "name": "Google Maps"
+        },
+        "google_pubsub": {
+          "config_flow": false,
+          "iot_class": "cloud_push",
+          "name": "Google Pub/Sub"
         },
         "google_sheets": {
           "config_flow": true,
           "iot_class": "cloud_polling",
           "name": "Google Sheets"
+        },
+        "google_translate": {
+          "config_flow": false,
+          "iot_class": "cloud_push",
+          "name": "Google Translate Text-to-Speech"
+        },
+        "google_travel_time": {
+          "config_flow": true,
+          "iot_class": "cloud_polling"
+        },
```

Comment:
@emontnemery why doesn't this integration have a name?

Reply:
Translatable name

Reply:
nm, it's translatble.